.github/workflows/detect-breaking-changes-levitate.yml

```diff
@@ -154,26 +158,16 @@ jobs:
           project_id: 'grafanalabs-global'
           install_components: 'bq'
 
-      - name: Get link for the Github Action job
```

Comment:
we don't need this job as-is anymore, a general link to the levitate pipeline is now enough since we have all in markdown